test/browser/render.test.js

```diff
@@ -91,6 +91,13 @@ describe('render()', () => {
 		expect(scratch.childNodes[0].nodeName).to.equal('SPAN');
 	});
 
+	it('should create input tag with type date', () => {
+		render(<input type="date" />, scratch);
+		expect(scratch.childNodes).to.have.length(1);
+		expect(scratch.childNodes[0].nodeName).to.equal('INPUT');
+		expect(scratch.childNodes[0].type).to.equal('date');
```

Comment:
I think we should test something that more closely resembles the original bug report. The fix in this PR specifically addresses the case where setting `input.date` would throw an error in IE11. The test case should reflect only that. On our CI our test suite will be run in various browsers, including IE11. So we only need to check if it throws or not.

```js
it('should not throw in IE11 with type date', () => {
  expect(() => render(<input type="date" />, scratch)).to.not.throw();
});
```